weather_mv/loader_pipeline/ee.py

```diff
@@ -459,15 +458,16 @@ def convert_to_asset(self, queue: Queue, uri: str):
                 attrs = ds.attrs
                 data = list(ds.values())
                 asset_name = get_ee_safe_name(uri)
-                channel_names = [da.name for da in data]
-                start_time, end_time, is_normalized = (attrs.get(key) for key in
-                                                       ('start_time', 'end_time', 'is_normalized'))
+                channel_names = list(self.band_names_dict.values()) \
```

Comment:
`band_names_dict`'s values would not always be in order.
i.e. data has 2 data_vars ['precip', 'u10']. And if the user gives band_names_dict = '{"u10": "new_u10", "precip": "new_precip"}'.

Reply:
Could we instead ask the user to input the `band_names` flag as a list of the band names (in order) instead of a dictionary? Because anyhow the mapping is something like this -
```
{
  'b0': 'band_name_1',
  'b1': 'band_name_2'
}
```

Reply:
Resolved